website/blog/modules/ROOT/pages/15-android-build-flow.adoc

```diff
@@ -0,0 +1,223 @@
+= How does Mill build Android apps?
+
+// tag::header[]
+:author: Vasilis Nicolaou
+:revdate: 29 August 2025
+
+_{author}, {revdate}_
+
+
+Until recently, Gradle was the only realistic option for Android builds. Today, Mill - a fast, predictable build tool for Java, Scala, and Kotlin - can build Android applications end-to-end.
```

Comment:
Let's leave out `fast, predictable` here. Whether or not it's true, it sounds too salesly. Let's just say "Mill provides an alternative to Gradle that is easier to use and learn", and hope that enough people have had trouble with Gradle that they empathize